pkg/distro/defs/rhel-10/distro.yaml

```diff
@@ -837,6 +838,9 @@ image_types:
 
   "vagrant-libvirt": &vagrant_libvirt
     <<: *qcow2
+    # we have to reset the aliases otherwise this type
+    # will inherit the name aliases causing a conflict
+    name_aliases: []
```

Comment:
I was unsure whether to do this or maybe redefine a new base type? Will leave this open for comments & suggestions